Sofee.Core.Api/Brokers/Storages/StorageBroker.Countries.cs

```diff
@@ -4,12 +4,27 @@
 // -------------------------------------------------------
 
 using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.ChangeTracking;
 using Sofee.Core.Api.Models.Countries;
+using System.Threading.Tasks;
 
 namespace Sofee.Core.Api.Brokers.Storages
 {
     public partial class StorageBroker
     {
         public DbSet<Country> Countries { get; set; }
+        
+        public async ValueTask<Country> InsertCounrtyAsync(Country country)
+        {
+            using var broker = new StorageBroker(this.configuration);
+
+            EntityEntry<Country> countryEntityEntry =
+                await broker.Countries.AddAsync(country);
+
+            await broker.SaveChangesAsync();
+
+            return countryEntityEntry.Entity;
+        }
     }
+
```

Comment:
@hoxhaleo319 can you please remove this empty line?

Reply:
@ElbekDeveloper I removed the empty line.

Reply:
Good job! Waiting to become contributor so that I can merge the changes @hoxhaleo319...